x-pack/plugins/synthetics/server/legacy_uptime/lib/saved_objects/saved_objects.ts

```diff
@@ -23,6 +24,7 @@ export const registerUptimeSavedObjects = (
   isServiceEnabled: boolean
 ) => {
   savedObjectsService.registerType(umDynamicSettings);
+  savedObjectsService.registerType(privateLocationsSavedObject);
 
   if (isServiceEnabled) {
```

Comment:
We need to reconsider how we're handling this boolean to ensure that monitor saved objects are registered off-cloud. Currently this boolean is `Booolean(this.server.config.service). On prem users would have no need to add the manifest url to their config settings, so we should probably remove this boolean.